.env.example

```
@@ -29,3 +35,10 @@ VITE_WHATSAPP_API_URL=http://localhost:8080
 VITE_WHATSAPP_API_USERNAME=admin
 VITE_WHATSAPP_API_PASSWORD=your_secure_password
 VITE_WHATSAPP_API_TIMEOUT=10000
+
+# WhatsPoints API (for development - uses client-side in dev mode)
+VITE_WHATSPOINTS_API_URL=http://localhost:8080
+VITE_WHATSPOINTS_API_USERNAME=admin
+VITE_WHATSPOINTS_API_PASSWORD=your_dev_password
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_WHATSPOINTS_API_PASSWORD key should go before the VITE_WHATSPOINTS_API_URL key

(UnorderedKey)